src/meta/src/manager/env.rs

```diff
@@ -81,6 +81,9 @@ pub struct MetaOpts {
     /// Whether run in compaction detection test mode
     pub compaction_deterministic_test: bool,
 
+    /// Start id of SST table file. See [`IdGeneratorManager`] for details.
+    pub sst_id_start: Option<u64>,
```

Comment:
For a cluster, this option is actually used only once when initialized first time and is ignored later. Can we avoid this option?

Reply:
I need to shift the start sst_id of the embedded Meta node to avoid id conflicts with the original Meta. Any alternatives?

Reply:
Call `get_new_sst_ids` to explicitly fetch then throw away certain sst ids?

Reply:
Hmm.. I don't get it. Do you mean we should add this logic to the compactor when runs in the deterministic mode?

Reply:
Discussed offline, will fix it in the next PR.